helm-charts/auth-apisix/apis-crd-helm/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Identity provider OIDC config
+oidc:
+  realm: <your realm name>
+  client_id: <your oidc client id>
+  client_secret: <your oidc client secret>
+  discovery: <your oidc discovery endpoint>
+  introspection_endpoint: <your oidc introspection endpoint>
+
+# APISIX chatqna api config
+chatqna:
+  namespace: <namespace in which your chatqna service is running>
+  hostname: <hostname for external access>
```

Comment:
what does this hostname for external access mean?

Reply:
added more details for clarity